action.yml

```diff
@@ -1,32 +1,80 @@
-name: "Lazy Action Nodejs"
-description: "Github action to perform build, test , scan and generate image."
+---
+# yamllint disable rule:line-length
+name: Lazy Action Nodejs
+description: |
+  Github Action to perform build, test , scan and generate Image for Python
+
+  ## Permissions
+
+  Add the following permissions to the job
+
+  ```yaml
+  permissions:
+    id-token: write
+    contents: read
+  ```
+
+  ## Usage
+
+  ```yaml
+      - name: Actions Nodejs
+        uses: variant-inc/actions-nodejs@v2
+        with:
+          ecr_repository: 'demo/example'
+          nodejs-version: 18
+  ```
 inputs:
   dockerfile_dir_path:
-    description: "Directory path to the dockerfile"
+    description: Directory path to the dockerfile
     required: false
+    default: .
   npm_test_script_name:
-    description: "npm test script name"
+    description: npm test script name
     required: false
-    default: "test"
-  github_token:
-    description: "Github token"
-    required: true
-  container_push_enabled:
-    description: "Enable Build and Push Container Image"
-    required: false
-    default: "true"
+    default: test
   ecr_repository:
-    description: "Ecr repository name"
-    required: false
-  sonar_project_key:
-    description: "When using a monorepo, you may need to specify the project key"
+    description: Ecr repository name
+    required: true
```

Comment:
I believe this shoud be set as `false` simmilar like in actions-python, to enable testing the image without push to ECR.
https://github.com/variant-inc/actions-python/pull/23/files#diff-1243c5424efaaa19bd8e813c5e6f6da46316e63761421b3e5f5c8ced9a36e6b6R31